torch/_inductor/kernel/mm_common.py

```diff
@@ -134,7 +139,7 @@ def filtered_configs(
 mm_kernel_configs = (
     [
         {"config": (32, 32, 16, 1, 2), "cond": True},
-        {"config": (32, 32, 128, 2, 4), "cond": torch.version.hip is None},
+        {"config": (32, 32, 128, 2, 4), "cond": True},
```

Comment:
Curious: this config (and the two below) didn't work before but are going to work now? Also with the old Triton version?

Reply:
We disabled these a long time ago as we were seeing runtime errors when we would throw a shmem OOM error. The autotune implementation seems safe enough now to not completely error out if we hit a config that throws oom issues for us. The CI is green here on the older triton still so I think we're safe here.